### ISSUE TITLE
[Bug]: station messages wrapped with <p> </p>

### ISSUE THREAD

**Issue by ReX1983** [crismc/homeassistant_nationalrailtimes_integration#3]
### Requirements

- [X] I have updated NationalRailTimes to the latest available version
- [X] I did a search to see if there is a similar issue.

### Current Behavior

Some station messages are reported as:

<p>The Ticket Office is currently closed at XYZ station.</p>

### Expected Behavior

<p> and </p> should not be displayed.

### Steps To Reproduce

You just need a station that has ticket office closed message.

### Environment

```markdown
- Browser:
- HA Version:
- NationalRailTimes:
```

### Anything else?

I have modified my local copy of ` def extra_state_attributes(self)` adding the followings before return:
```
        pattern1 = re.compile(r'</p>')
        attributes["message"] = [pattern1.sub('', sub) for sub in attributes["message"]]
        
        pattern2 = re.compile(r'<p>')
        attributes["message"] = [pattern2.sub('', sub) for sub in attributes["message"]]
```

It works, would be good to add something similar in your implementation.

**Comment by crismc** [comment]
Yeah, did see this every now and again on mine too... waiting for it to pop back up so I can test, and will look to add a wrapper.

Will probably look to add it in the apidata.py/message method, as sensor.py/extra_state_attributes was designed as more of a controller / gatherer of data than a manipulator. Although if you've already issued a fix locally, happy to review a pull request and merge it in, with full crediting to you.

Honestly, nice to see someone other than me is getting use out of this :)

**Comment by Macka1993** [comment]
I'm also experiencing this bug, it causes the lovelace card to disappear from my dashboard, I've worked around it by choosing another local station

**Comment by eveiscoull** [comment]
Looks like I'm facing this issue too on my side. We've got some pretty hellish weather at the moment here in Scotland, so the alerts will likely be on for a while longer if you want to test with mine :)

![image](https://github.com/crismc/homeassistant_nationalrailtimes_integration/assets/63597985/46254b0b-27b2-44fd-9772-21105152cba5)

**Comment by ID-Ant** [comment]
So not sure if this got resolved, but just seen this on my card as well:

<img width="314" alt="Screenshot 2024-11-28 at 08 50 53" src="https://github.com/user-attachments/assets/04deac27-48c5-4c26-957b-14824c2a3aa4">



Message
<p>Trains between Rock Ferry and Hooton may be cancelled, delayed by up to 20 minutes or revised. Latest information can be found in <a href="https://www.nationalrail.co.uk/service-disruptions/spital-20241128/">Status and Disruptions.</a></p>
